pkg/operations/operations_aws.go

```diff
@@ -71,6 +84,14 @@ func AWSOperationsProvider(
 	return prov, nil
 }
 
+func getPropertyMapStringValue(m resource.PropertyMap, k resource.PropertyKey) string {
+	v, ok := m[k]
+	if !ok {
+		return ""
+	}
+	return v.StringValue()
```

Comment:
StringValue panics if it isn't a string. This should probably check IsString first.